### ISSUE TITLE
Confusion regarding training using khoya_ss gui

### ISSUE THREAD

**Issue by dr-dark-flames** [KohakuBlueleaf/LyCORIS#28]
```
{
  "pretrained_model_name_or_path": "B:/Apps/Stable Diffusion/stable-diffusion-webui/models/Stable-diffusion/Anything_V4.5.safetensors",
  "v2": false,
  "v_parameterization": false,
  "logging_dir": "B:/Apps/Stable Diffusion/kohya_ss/data/Dehya_LoRA/log",
  "train_data_dir": "B:/Apps/Stable Diffusion/kohya_ss/data/Dehya_LoRA/image",
  "reg_data_dir": "",
  "output_dir": "B:/Apps/Stable Diffusion/kohya_ss/data/Dehya_LoRA/model",
  "max_resolution": "768,768",
  "learning_rate": "0.0001",
  "lr_scheduler": "constant",
  "lr_warmup": "0",
  "train_batch_size": 6,
  "epoch": 10,
  "save_every_n_epochs": 1,
  "mixed_precision": "bf16",
  "save_precision": "bf16",
  "seed": "",
  "num_cpu_threads_per_process": 4,
  "cache_latents": true,
  "caption_extension": "",
  "enable_bucket": true,
  "gradient_checkpointing": false,
  "full_fp16": false,
  "no_token_padding": false,
  "stop_text_encoder_training": 0,
  "xformers": true,
  "save_model_as": "safetensors",
  "shuffle_caption": true,
  "save_state": false,
  "resume": "",
  "prior_loss_weight": 1.0,
  "text_encoder_lr": "5e-5",
  "unet_lr": "0.0001",
  "network_dim": 32,
  "lora_network_weights": "",
  "color_aug": false,
  "flip_aug": false,
  "clip_skip": 2,
  "gradient_accumulation_steps": 1.0,
  "mem_eff_attn": false,
  "output_name": "Dehya",
  "model_list": "custom",
  "max_token_length": "150",
  "max_train_epochs": "",
  "max_data_loader_n_workers": "",
  "network_alpha": 1,
  "training_comment": "",
  "keep_tokens": "0",
  "lr_scheduler_num_cycles": "",
  "lr_scheduler_power": "",
  "persistent_data_loader_workers": false,
  "bucket_no_upscale": true,
  "random_crop": false,
  "bucket_reso_steps": 64.0,
  "caption_dropout_every_n_epochs": 0.0,
  "caption_dropout_rate": 0,
  "optimizer": "DAdaptation",
  "optimizer_args": "",
  "noise_offset": "",
  "LoRA_type": "LyCORIS/LoHa",
  "conv_dim": 32,
  "conv_alpha": 1,
  "sample_every_n_steps": 0,
  "sample_every_n_epochs": 0,
  "sample_sampler": "euler_a",
  "sample_prompts": "",
  "additional_parameters": ""
}
```
Can anybody explain to me what is the difference between network rank and conv dim??
Same thing with conv alpha and normal alpha..I dont understand how would i set them up and please if  u would be so kind and point out any mistake ive done in those configuration as im a newbie.
Thanks in advance!

**Comment by KohakuBlueleaf** [comment]
network dim is for linear layer
conv dim is for conv layer

network alpha is for linear layer
conv alpha is for conv layer

**Comment by dr-dark-flames** [comment]
yeah but how do i set the training for loha? do i need to set both values and what would the values be?

> network dim is for linear layer conv dim is for conv layer
> 
> network alpha is for linear layer conv alpha is for conv layer

**Comment by KohakuBlueleaf** [comment]
@dr-dark-flames 
How do i set the training for loha: depends on what training script you use. For kohya-ss, use --network_args(see readm). For others, I don't know.

Do I need to set both value: no, you can only set the network_dim/alpha, and conv part will automatically use them.

What would values be: 
dim: 2~16
alpha: recommend 1 (or any fixed value)

**Comment by dr-dark-flames** [comment]
Thank you brother!!!
